src/examples/talos.cpp

```diff
@@ -59,6 +59,8 @@ int main()
             Eigen::VectorXd commands = (init_positions + delta_pos) - robot->positions(dofs);
             robot->set_commands(commands, dofs);
         }
+        simu.set_text_panel(robot->model_filename() + "\n"
+            + simu.default_text_panel());
```

Comment:
It is a bit weird to have to re-set the text just for the simulation time to be updating. We could add a boolean (whether to display the default text or not), and it should be added automatically. What do you think?

Reply:
Why not. I kinda like the idea to "make your own" : here you can for instance add a title above the default (if you do not call set_text_panel, then you have the default). But I am fine with a Boolean if you prefer.